.github/workflows/ci.yml

```diff
@@ -139,14 +164,18 @@ jobs:
     - uses: actions/checkout@v4
       with:
         submodules: true
-
+    
     # - name: Initialize CodeQL
     #   if: matrix.config.python-version == '3.10'
     #   uses: github/codeql-action/init@v3
     #   with:
     #     languages: python, cpp
     #     setup-python-dependencies: false
 
+    - name: Set up QEMU (Linux ARM64)
+      if: matrix.config.os-name == 'linux' && matrix.config.python-arch == 'aarch64'
+      uses: docker/setup-qemu-action@v2
```

Comment:
I noticed that the action already has a `v3` out, any reason you didn't use that? If not, let's update to the latest version.